sdk/attestation/azure-security-attestation/CHANGELOG.md

```diff
@@ -5,12 +5,15 @@
 ### Features Added
 
 ### Breaking Changes
+* Added `TpmAttestationResult` model
+* Changed `attestTpm` parameter from str to bytes to match `attestOpenEnclave` and `attestSgxEnclave`
```

Comment:
We cannot break the existing API signature/behaviour.
If there's an exact correlation between input and output, we can possibly add overloads, otherwise, if the previous implementation should no longer be supported for some reason, then we would need to deprecate it and introduce a new API.

Reply:
Are we following semantic versioning for this SDK? If so, in this case can't we just bump up the major version to accommodate this breaking change?

Reply:
We do not do major version bumps unless there is no choice. Put simply, we would only ever introduce a breaking change if there was a major risk in not doing so - e.g. a security vulnerability, data loss etc.

Reply:
Currently no production use of attestTpm as far as we can tell.